django_ckeditor_5/static/django_ckeditor_5/app.js

```diff
@@ -26,22 +26,27 @@ function getCookie(name) {
 /**
  * Checks whether the element or its children match the query and returns
  * an array with the matches.
- * 
+ *
  * @param {!HTMLElement} element
  * @param {!string} query
- * 
+ *
  * @returns {array.<HTMLElement>}
  */
 function resolveElementArray(element, query) {
-    return element.matches(query) ? [element] : [...element.querySelectorAll(query)];
+    try {
+        return element.matches(query) ? [element] : [...element.querySelectorAll(query)];
+    } catch (err) {
+        console.warn(err)
+        return [element]
```

Comment:
from what I've understand we should probably return an empty array here if no element was found